dashboard/config/routes.rb

```diff
@@ -893,5 +893,7 @@
 
   get '/backpacks/channel', to: 'backpacks#get_channel'
 
-  resources :reviewable_projects, only: [:create, :destroy]
+  resources :reviewable_projects, only: [:create, :destroy] do
+    get :reviewable_status, on: :collection
```

Comment:
I think `on: :collection` is for routes that would return multiple objects, but I think this is just aimed at finding a single object? I think just remove the `on: :collection` argument -- more details here:

https://guides.rubyonrails.org/routing.html#adding-collection-routes

Reply:
Ah okay, let me try that - my understanding was that without `on: :collection` the route would have a reference a specific ID, i.e. reviewable_projects/:id (similar to delete) whereas the record may not exist for this request? But not too sure about that behavior, I'll take a look at this.

Reply:
Yeah, so removing `on: :collection` seems to mandate an ID in the url since it's defined inside of the `resources` block I believe; but defining `get 'reviewable_projects/reviewable_status', to: 'reviewable_projects#reviewable_status'` outside of the block seems to work as intended - does that look okay?

Reply:
Ah got it -- yeah, I think outside of the block makes more sense than marking it as a collection route.